### PR TITLE
refactor[script]: Included nfs based jobs for cstor and jiva

### PR DESCRIPTION
**Pull request by somesh2905**
Signed-off-by: somesh kumar <somesh.kumar@mayadata.io>

1. Included nfs based jobs for cstor and jiva

### CODE REVIEW

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0YB8-nfs-cstor-volume-mgmt/nfs-cstor-volume-mgmt [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+#Creating e2e custom resource result for the test.
+bash Openshift-EE/utils/pooling jobname:nfs-cstor-istgt-kill
+bash Openshift-EE/utils/e2e-cr jobname:nfs-cstor-volume-mgmt-kill jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag"
+
+################
+# LitmusBook 1 #
+################
+# Performing Application Container Chaos
+
+run_id="mgmt-kill";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=openebs-target-failure metadata=${run_id})
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0YB8-nfs-cstor-volume-mgmt/nfs-cstor-volume-mgmt]: include nfs also in run_id mgmt-kill-nfs

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0ACF-nfs-app-kill/nfs-app-kill [hunk 2]
```diff
@@ -24,122 +24,29 @@ echo $present_dir
 #Creating e2e custom resource result for the test.
 bash Openshift-EE/utils/e2e-cr jobname:nfs-app-kill jobphase:Waiting
 bash Openshift-EE/utils/e2e-cr jobname:nfs-app-kill jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag"
-
+bash Openshift-EE/utils/e2e-cr jobname:nfs-target-kill jobphase:Waiting
+bash Openshift-EE/utils/e2e-cr jobname:nfs-cstor-istgt jobphase:Waiting
+bash Openshift-EE/utils/e2e-cr jobname:nfs-cstor-volume-mgmt jobphase:Waiting
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0ACF-nfs-app-kill/nfs-app-kill]: check the job names

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0ACF-nfs-app-kill/nfs-app-kill [hunk 3]
```diff
@@ -24,122 +24,29 @@ echo $present_dir
...
--e 's/value: docker/value: cri-o/g' \
--e 's/value: app-jenkins-ns/value: app-nfs-ns/g' nfs_app_kill.yml
+sed -i -e '/name: NFS_NAMESPACE/{n;s/.*/            value: app-nfsv4-ns/}' \
+-e '/name: NFS_LABEL/{n;s/.*/            value: app=nfs/}' \
+-e '/name: APP_LABEL/{n;s/.*/            value: name=minio/}' \
+-e '/name: APP_NAMESPACE/{n;s/.*/            value: app-minio-ns/}' \
+-e '/name: CONTAINER-RUNTIME/{n;s/.*/            value: cri-o/}' nfs_app_kill.yml
 
-echo "Running the litmus test for Busybox Deployment application pod kill.."
+echo "Running the litmus test for minio Deployment application pod kill.."
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0ACF-nfs-app-kill/nfs-app-kill]: remove this echo command

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+-e '/name: NFS_LABEL/{n;s/.*/            value: app=nfs/}' \
+-e '/name: NFS_PVC/{n;s/.*/            value: nfs-pvc-claim/}' \
+-e '/name: APP_LABEL/{n;s/.*/            value: name=minio/}' \
+-e '/name: APP_NAMESPACE/{n;s/.*/            value: app-minio-ns/}' nfs_target_kill.yml
+
+sed -i -e 's/value: "target-zrepl-kill"/value: "target-kill"/g' \
+-e 's/value: "cstor-volume-mgmt"/value: "cstor-istgt"/g' \
+-e 's/value: docker/value: cri-o/g' nfs_target_kill.yml
+
+echo "Running the litmus test for minio Deployment application Target kill.."
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill]: remove this echo command

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/I7QP-nfs-cstor-istgt/nfs-cstor-istgt [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+#Creating e2e custom resource result for the test.
+bash Openshift-EE/utils/pooling jobname:nfs-target-kill
+bash Openshift-EE/utils/e2e-cr jobname:nfs-cstor-istgt-kill jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag"
+
+################
+# LitmusBook 1 #
+################
+# Performing Application container Chaos
+
+run_id="istgt-kill";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=openebs-target-failure metadata=${run_id})
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/I7QP-nfs-cstor-istgt/nfs-cstor-istgt]: include nfs in run_id

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+cp experiments/chaos/nfs_chaos/nfs_openebs_target_failure/run_litmus_test.yml nfs_target_kill.yml
+
+sed -i -e '/name: NFS_NAMESPACE/{n;s/.*/            value: app-nfsv4-ns/}' \
+-e '/name: NFS_LABEL/{n;s/.*/            value: app=nfs/}' \
+-e '/name: NFS_PVC/{n;s/.*/            value: nfs-pvc-claim/}' \
+-e '/name: APP_LABEL/{n;s/.*/            value: name=minio/}' \
+-e '/name: APP_NAMESPACE/{n;s/.*/            value: app-minio-ns/}' nfs_target_kill.yml
+
+sed -i -e 's/value: "target-zrepl-kill"/value: "target-kill"/g' \
+-e 's/value: "cstor-volume-mgmt"/value: "cstor-istgt"/g' \
```
Comment (nsathyaseelan) [inline Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill]: change this target-kill into target-delete  and remove `-e 's/value: "cstor-volume-mgmt"/value: "cstor-istgt"/g' \
 `.

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0YB8-nfs-cstor-volume-mgmt/nfs-cstor-volume-mgmt (hunk 1 above)
Comment (somesh2905) [reply Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0YB8-nfs-cstor-volume-mgmt/nfs-cstor-volume-mgmt]: included

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0ACF-nfs-app-kill/nfs-app-kill (hunk 2 above)
Comment (somesh2905) [reply Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0ACF-nfs-app-kill/nfs-app-kill]: resolved

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0ACF-nfs-app-kill/nfs-app-kill (hunk 3 above)
Comment (somesh2905) [reply Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/0ACF-nfs-app-kill/nfs-app-kill]: removed

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill (hunk 4 above)
Comment (somesh2905) [reply Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill]: removed

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/I7QP-nfs-cstor-istgt/nfs-cstor-istgt (hunk 5 above)
Comment (somesh2905) [reply Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/I7QP-nfs-cstor-istgt/nfs-cstor-istgt]: included

File: Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill (hunk 6 above)
Comment (somesh2905) [reply Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/3UFA-nfs-target-kill/nfs-target-kill]: resolved

**Pull request merged**